### DISCUSSION TITLE
Powershell script won't close after running

### DISCUSSION THREAD

**Discussion by tmcder24** [PowerShell/PowerShell#16790] [Q&A]
Hey everyone, I'm super new to scripting with Powershell.  Working on a script for my company for us to remotely back up Chrome Bookmarks.  The script works, but then it just stays open and stays in a "running" state.  I tested adding a few lines after the initial script, it looks like it doesn't move to that either, so it's getting hung up for some reason after doing what it's supposed to do.  Any suggestions would be awesome

$ComputerName = read-host "Input Target Computer"
$username = read-host "Input Target Username"
get-childitem -Path \\$ComputerName\c$\Users\$username\Appdata\Local\Google\Chrome\ -Recurse | 
    Where-object Name -eq "Bookmarks" | 
        copy-item -Destination \\domain\common\home\$username\

**Comment by mnaray** [comment]
You might have set the noexit attribute to true somehow permanently. This could cause the CLI to stay open, even after the actual script was run.
Could you describe how you usually run your script, if you can't fix it with the noexit solution?

**Comment by kasini3000** [comment]
I wrote a powershell automated batch framework that seems to solve your problem. But it doesn't seem like it should be discussed here.
Would you like to resubmit the issue at here？
https://github.com/kasini3000/kasini3000/issues/new/choose
